sqldelight-gradle-plugin/src/test/fixtures/duplicate-table-name/failure.txt

```diff
@@ -0,0 +1,5 @@
+Table2.sq line 1:13 - Table already defined with name test
+  1		CREATE TABLE test (
+   		             ^^^^
+  2		  _id LONG NOT NULL PRIMARY KEY
+  3		)
```

Comment:
so one thing that super sucks about these files is you have to use tabs to make them and even then that didn't work, i had to copy/paste the output i got from running the command line. I'll probably just make these use spaces in a follow up PR.